src/tests/editor.test.js

```diff
@@ -1781,10 +1781,10 @@ describe('Editor', () => {
                     ),
                     stepFunction: deleteBackward,
                     contentAfter: unformat(
-                        `<p>a[]</p>
+                        `<p>a[b</p>
                         <table><tbody>
-                            <tr><td><br></td><td><br></td></tr>
-                            <tr><td>h</td><td>ij</td></tr>
+                            <tr><td>cd</td><td>ef</td></tr>
+                            <tr><td>g]h</td><td>ij</td></tr>
```

Comment:
This is nonsense and I'm pretty sure that you know it. Imaging in Excel you couldn't select multiple cells and just backspace them away ? I understand that it is easier in your implementation if you do not have to deal with unbreakables farther than "woops, unbreakables, stop it all", but we're loosing useful behavior here IMO.

Reply:
Same reply.